project/cmake/README.md

```diff
@@ -168,13 +221,36 @@ scanning them would take up an unreasonable amount of configure time.
 When using the makefile builds a few extra targets are defined:
 
 - `make check` builds and executes the test suite.
+- `make check-valgrind` builds and executes the test suite with valgrind memcheck.
 
 Code coverage (with Gcov, LCOV and Gcovr) can be built on Linux:
 
 - CMake has to be executed with `-DCMAKE_BUILD_TYPE=Coverage`
 - `make coverage` generates an HTML code coverage report.
 - `make coverage_xml` generates an XML code coverage report.
 
+## Building binary addons
```

Comment:
what audience is this targeted to? developers, in particular developers who build new addons, need to build this differently.
this doc should stress the pitfall: if you built addon 1, you can't build addon 2 unless deleting the bootsstrap dir first.